LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014 ClusterHQ
```

Comment:
The documentation at this point is weird - it says:

```
Ensure copyright dates in LICENSE are up-to-date:

XXX: Process to be decided. See https://github.com/ClusterHQ/flocker/issues/525
```

However that is a related issue closed as "Done". Either create an issue to be linked to at this part of the docs or update them to say something like "If no releases have been finished earlier in this calendar year, add this year to the LICENSE file". I'll work on the assumption that what you have done here is right.